### PR TITLE
FLUID-5816: Added content about web games and sims.

### PR DESCRIPTION
**Pull request by jhung**
@acheetham can you take a look at this? If it's good, then the document "Web Games and Simulations"  should be mentioned in the FLOE-438 pull request: https://github.com/fluid-project/floeproject.org/pull/38

### CODE REVIEW

**Comment by jhung** [comment]
I'm adding links to MathML Cloud and Poet Image Description Tool. Do not merge this just yet.

File: site-structure.json [hunk 1]
```diff
@@ -34,7 +34,8 @@
                     { "pageName": "Video Content and Learning", "href": "/VideoContentAndLearning.html"},
                     { "pageName": "Audio Content and Learning", "href": "/AudioContentAndLearning.html"},
                     { "pageName": "Authoring of Content", "href": "/AuthoringOfContent.html"},
-                    { "pageName": "Cognitive Considerations", "href": "/CognitiveConsiderations.html"}
+                    { "pageName": "Cognitive Considerations", "href": "/CognitiveConsiderations.html"},
+                    { "pageName": "Web Games and Simulations", "href": "/WebGamesAndSimulations.html"}
```
Comment (acheetham) [inline site-structure.json]: I think this ToC link to the games/sims page should be moved up after audio content and before authoring. It seems geared toward a particular type of content, like video, audio, etc) and so might make more sense near those topics. What do you think, @jhung?

File: src/documents/WebGamesAndSimulations.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
+---
+title: Web Games and Simulations
+layout: default
+category: Methods
+---
+
+Interactive games can provide rich learning experiences for many students - but for some, these games are a accessibility and inclusivity challenge. This guide aims at giving some practical techniques to help improve the inclusiveness and accessibility of interactive content whether retrofitting existing content or creating a new game from scratch.
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: "a" should be "an" in "these games are a accessibility and inclusivity challenge"

File: src/documents/WebGamesAndSimulations.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+**Scope of this Guide:** This guide deals with interactive games built using standard web technologies such as HTML5, CSS, SVG, and JavaScript. By using established web standards, the content you create has a higher chance of being understood by assistive technologies.
+
+## Orienting the User
+
+Regardless of the modality used for interaction, a logical, predictable structure helps users focus on learning and the experience instead of getting stuck on the interaction. Consistency and predicability is important.
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: I think this sentence might flow better – and still say the right thing – if you remove the phrase "and the experience" after "focus on learning"

File: src/documents/WebGamesAndSimulations.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+Help orient the user by:
+
+* having a consistent structure.
+* describe the scene and setting so non-visual users can understand their context.
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: "describing" instead of "describe"

File: src/documents/WebGamesAndSimulations.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+* identifying the actors and interactive parts.
+* giving meaningful and predictable landmarks for the user.
+* use progress and status indicators as necessary.
+* try to keep the structure stable. Be mindful about removing items completely from the structure - consider disabling elements rather than removing them.
+
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: "using" and "trying" for the last two bullet points, instead of "use" and "try"

Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: End each bullet point with a semi-colon. Reserve the period for the last bullet point.

File: src/documents/WebGamesAndSimulations.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+## Player Choices and Controls
+
+Some people make choices easily, while others require more time before making decisions. Choices in games can be exciting for some, but stressful for others.
+
+If presenting learners with multiple choices that affect the outcome of a game:
+
+* If possible, make choices forgiving: give opportunity for users to recover from or change their choices.
+* Avoid player choices which require decisions within a time limit or provide options to disable such features.
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: Add a comma before "or"

File: src/documents/WebGamesAndSimulations.md [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+
+Other best practices are:
+* Label each button and control. Labels should use simple terminology and define any terms if necessary.
+* Labels should state simply what the button will do. If longer descriptions are needed, use an `aria-describedby` property.
+* Buttons, links, controls should make it clear what the outcome would be. For example: A choice called "I'm out of here" could be changed to "I'm out of here - quit game", or simply "Quit".
+* In some cases text where text labels are not visible (example: icon buttons that do not have visible text labels), consider using this technique for making invisible content for screen readers: <a href="http://webaim.org/techniques/css/invisiblecontent/" rel="nofollow" target="_blank" class="link-external">"Invisible Content Just for Screen Reader Users" on WebAim.org</a>.
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: This opening phrase – "In some cases text where text labels are not visible" is a bit awkward. I'm not quite sure what you're trying to say...

Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: In the brackets, perhaps "for example," instead of "example:"

Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: Instead of "consider using this technique for making invisible content for screen readers: "Invisible Content Just for Screen Reader Users" on WebAim.org", perhaps just "consider using the technique described in "Invisible Content Just for Screen Reader Users" on WebAim.org"

File: src/documents/WebGamesAndSimulations.md [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+
+## Communicating Events and Actions with Text and Sound
+
+A large part of interactive content are the events that happen during and after a user performs an action. Often these occur as on-screen visuals, animations, and transitions. The following are some techniques for creating an equivalent experience for non-sighted learners:
+
+* Use ARIA live-regions to communicate changes to the interface.
+* Use ARIA `describedby` to write text descriptions of what has happened.
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: These mentions of ARIA techniques might benefit from links to relevant articles explaining the techniques?

File: src/documents/WebGamesAndSimulations.md [hunk 9]
```diff
@@ -0,0 +1,164 @@
...
+* Consider use of various sounds to convey meaning. For example, to convey proximity of one object to another, use a tone of increasing pitch like a Theremin (visit <a href="https://en.wikipedia.org/wiki/Theremin" rel="nofollow" target="_blank" class="link-external">Theremin on Wikipedia</a>).
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: https://www.youtube.com/watch?v=BzEHItrTMHM :-)

File: src/documents/WebGamesAndSimulations.md [hunk 10]
```diff
@@ -0,0 +1,164 @@
...
+* Use WAI-ARIA to improve accessibility. See: <a href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA" rel="nofollow" target="_blank" class="link-external">Mozilla Developer ARIA - good resource for learning, examples, blogs</a>.
+
+## Visuals
+
+* Provide text descriptions for important visuals: images should have alt-text and longer text descriptions if necessary. Use `aria-describedby` or `longdesc` as required. See: <a href="http://www.w3.org/WAI/tutorials/images/complex/" rel="nofollow" target="_blank" class="link-external">http://www.w3.org/WAI/tutorials/images/complex/</a>
+* Content contained in the background (using background CSS property), :before or :after elements are ignored by screen readers. Do not put important content within these elements. Use standard elements like `<image>`, `<div>`, `<figure>`, etc. as needed.
+* Images which are in the background should be cosmetic and not critical to the outcomes. Use `<img>` and `<figure>` as necessary for important graphics / images.
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: These two bullet points about content in the background seem to be saying the same thing?

File: src/documents/WebGamesAndSimulations.md [hunk 11]
```diff
@@ -0,0 +1,164 @@
...
+* Unicode characters are often omitted by screen readers. Ellipsis (...), arrows, and other characters are ignored.
+* Acronyms and abbreviations should use the `<abbr>` element with the relevant aria properties. For example:
+    * `<abbr role="text" title="kilograms" aria-label="kilograms">kg</abbr>`
+    * `<abbr role="text" title="miles per hour" aria-label="miles per hour">mph</abbr>`
+
+##  Charts and Graphs
+
+Charts and graphs are an excellent way to convey data in a meaningful way. However, charts and graphs typically favour a student who is capable of perceiving the content and can cognitively process the information.
+
+Some methods in providing a more inclusive experience of charts and graphs:
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: Some methods "of"? instead of "in"

File: src/documents/WebGamesAndSimulations.md [hunk 12]
```diff
@@ -0,0 +1,164 @@
...
+
+* Provide alternatives to charts. Give text descriptions and interpretations of the data, give a table of data.
+* A good description identifies:
+    * What is being shown (i.e. the purpose of the chart)
+    * Describe the values (relative sizes and differences between values).
+    * Describe any patterns to may help us understand the data.
+    * Summarize the trend / data pattern.
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: These bullet points should have a consistent grammatical structure.

File: src/documents/WebGamesAndSimulations.md [hunk 13]
```diff
@@ -0,0 +1,164 @@
...
+* Also see:
+    * PSU’s chart accessibility guide - <a href="http://accessibility.psu.edu/images/charts/" rel="nofollow" target="_blank" class="link-external">http://accessibility.psu.edu/images/charts/</a>
+    * W3C's guide on describing complex images - <a href="http://www.w3.org/WAI/tutorials/images/complex/" rel="nofollow" target="_blank" class="link-external">http://www.w3.org/WAI/tutorials/images/complex/</a>
+
+If using a table of data:
+
+* Follow best practices for table formatting (i.e. proper use of table headers, no table nesting). <a href="https://www.webaccessibility.com/best_practices.php?technology_platform_id=8" rel="nofollow" target="_blank" class="link-external">See Data Table Best Practices at WebAccessibility.com</a>
+* Any math, symbols, units, or acronyms defined. For additional information about making math accessible, see [Mathematics](Mathematics.html).
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: Change the form of this to the active "Define any math, symbols..." Perhaps same for next point? Put the "Simplify..." first?

File: src/documents/WebGamesAndSimulations.md [hunk 14]
```diff
@@ -0,0 +1,164 @@
...
+
+##  Cognitive
+
+Information should be presented in a clear concise manner, and where possible starting with general statements first with more detailed information following.
+
+* Instructions should be clear and concise, using a language that is appropriate.
+* If there is any possibility of confusion, make help readily available.
+* Avoid really dense content and interactions. Leverage semantic markup such as `<section>` elements with proper `<h1>` headings to help group content into navigable chunks.
+* Consider presenting information as needed, rather than presenting it all up front.
+* For example: Instead of printing instructions up front in plain text, instructions on how to use a control may be better presented when the user is hovering or focused on that control (like a tooltip). This helps reduce cognitive load and gives information only when it's needed.
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: This example seems to be related to the previous bullet point. Perhaps it should simply be a continuation of that same bullet point? or a sub-bullet?

File: src/documents/WebGamesAndSimulations.md [hunk 15]
```diff
@@ -0,0 +1,164 @@
...
+
+For related information, see: <a href="https://wiki.fluidproject.org/x/D4DpAg" rel="nofollow" target="_blank" class="link-external">Designing interfaces to meet the needs of users with cognitive disabilities</a>.
+
+##  Visual Perception
+
+* Do not rely on colour to convey meaning. If this is unavoidable, provide a text label or text descriptions.
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: This might be a bit more clear if it said "colour alone" instead of just "colour"

File: src/documents/WebGamesAndSimulations.md [hunk 16]
```diff
@@ -0,0 +1,164 @@
...
+* Avoid using transition / animation effects on content that is critical to the outcomes of the game. These effects may be missed or misunderstood.
+* Use opacity, transparency, alpha channels with caution. These effects may make it more difficult for some users to distinguish content from the background.
+
+##  Keyboard / Keypad / Single-switch Interaction
+
+Many users, not just users of screen readers, use the keyboard to interact with their computer. Some users may be using a keypad, a joystick mapped to keys, or a single-switch to navigate a virtual keyboard. A succinct keyboard interaction will help these learners focus on the content and not the mechanics of interaction.
+
+* Tab order should be consistent with existing user experiences and conventions. Generally tab order follows reading order for your region (i.e. left-to-right, top-to-bottom).
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: This "i.e. left-to-right" should probably be "e.g." instead

File: src/documents/WebGamesAndSimulations.md [hunk 17]
```diff
@@ -0,0 +1,164 @@
...
+* Use custom hotkeys with caution as many screen readers have their own keys related to navigation and interaction.
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: This will be more clear if you say "screen readers have their own hotkeys..."

File: src/documents/WebGamesAndSimulations.md [hunk 18]
```diff
@@ -0,0 +1,164 @@
...
+* Default browser focus styling is often too subtle to be noticeable by many users. It is recommended to use focus styling that is more noticeable by either using a thicker border and better colour contrast.
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: The "either" should probably be followed by an "or" instead of an "and"

File: src/documents/WebGamesAndSimulations.md [hunk 19]
```diff
@@ -0,0 +1,164 @@
...
+
+You want to get your users into the experience as quickly as possible:
+* If there is a group of navigation elements such as a site menu, navigation breadcrumb, or other common "header" features up front - use accessibility "skip" links to jump to the main content. Also using HTML 5 semantic markup such as `<nav>` and `<header>` will help screen readers identify content and be able to skip past them if needed.
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: This should be a comma instead of a dash after "up front"

File: src/documents/WebGamesAndSimulations.md [hunk 20]
```diff
@@ -0,0 +1,164 @@
...
+* To add a skip link to your content, see this article on WebAIM: <a href="http://webaim.org/techniques/skipnav/" rel="nofollow" target="_blank" class="link-external">http://webaim.org/techniques/skipnav/</a>
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: This link might be better embedded into the reference to skip links above, instead of being a full bullet point

File: src/documents/WebGamesAndSimulations.md [hunk 21]
```diff
@@ -0,0 +1,164 @@
...
+* Tab order should properly "cycle around" when tabbing through. Watch for focus traps or hidden elements which may break the game for users.
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: "cycle around" should probably be expanded a little bit, for clarity

File: src/documents/WebGamesAndSimulations.md [hunk 22]
```diff
@@ -0,0 +1,164 @@
...
+
+##  Images, Symbols, Math, Icons, and Glyphs
+
+Make sure that symbols, math notation, and other glyphs have the desired textual output. For example: 1/2 should read as “one half” and not “one slash two”.
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: This would be more clear if it said "textual output when read by a screen reader."

File: src/documents/WebGamesAndSimulations.md [hunk 23]
```diff
@@ -0,0 +1,164 @@
...
+
+* Consider using MathML for math notations. See example: [Mathematics](Mathematics.html).
+* Acronyms and abbreviations should use the <abbr> element with the relevant aria properties. For example:
+    * <abbr role="text" title="kilograms" aria-label="kilograms">kg</abbr>
+    * <abbr role="text" title="miles per hour" aria-label="miles per hour">mph</abbr>
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: this markup is actually being interpreted - needs code styling

File: src/documents/WebGamesAndSimulations.md [hunk 24]
```diff
@@ -0,0 +1,164 @@
...
+    * Also see:  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr rel="nofollow" target="_blank" class="link-external"
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: The attributes on this external link are just being rendered as plain text

File: src/documents/EPUB3Resources.md [hunk 25]
```diff
@@ -18,3 +18,5 @@ category: Inclusive EPUB 3
...
 14. Accessibility Screening Methodology Guidelines and Checklist by Daisy Consortium - <a rel="nofollow" target="_blank" class="link-external" href="http://www.daisy.org/accessibility-screening-methodology-guidelines-and-checklist.html">http://www.daisy.org/accessibility-screening-methodology-guidelines-and-checklist.html</a> - a checklist to help evaluate the accessibility features of a reader system
+15. Poet Image Description Tool - <a href="http://diagramcenter.org/development/poet.html" rel="nofollow" target="_blank" class="link-external">http://diagramcenter.org/development/poet.html</a>. A web-based tool for creating and providing guidelines to writing image descriptions for images in existing DAISY and EPUB books.
+16. MathML Cloud Tool - a href="http://mathmlcloud.org/" rel="nofollow" target="_blank" class="link-external">http://mathmlcloud.org</a>. A math conversion tool that creates both images and alternative descriptions of math equations on the fly.
```
Comment (acheetham) [inline src/documents/EPUB3Resources.md]: This link to the mathml cloud tool isn't rendering properly

File: src/documents/GraphicsAndMultimedia.md [hunk 26]
```diff
@@ -22,6 +22,12 @@ The key to making embedded media understood in a wide variety of situations is t
 * Provide captions in plain text as a transcript, or use the `<track>` element to add synchronized captions to audio and video.
 * Multiple `<track>` elements can be used to provide alternate audio tracks in different languages or to provide audio descriptions. Note that some platforms may not support `<track>` elements.
 
+## Poet Image Description Tool
+
+The Poet Image Description Tool is a web-based tool for creating and providing guidelines to writing image descriptions for images in existing DAISY and EPUB books. This tool has resources to help content authors create image descriptions following an extended NCAM guideline. There is also a math editor which transcribes math into readable format,
+
```
Comment (acheetham) [inline src/documents/GraphicsAndMultimedia.md]: The paragraph above ends with a comma

File: src/documents/Mathematics.html.md.handlebars [hunk 27]
```diff
@@ -103,3 +103,16 @@ Units and measurements poses their own unique challenge as units are often writt
...
    * `<abbr role="text" title="meters per second" aria-label="meters per second">m/s</abbr>`
+
+## MathML Cloud
+
+Benetech has developed a tool called <a href="https://www.mathmlcloud.org/" rel="nofollow" target="_blank" class="link-external">MathML Cloud</a> which allows content creators to convert math notation into readable text and other formats. MathML Cloud can take math notation in ASCII text, MathML or Latex notation, and output a text equivalent, PNG, SVG, and MML which can be embedded into EPUB 3 and HTML.
+
+<figure>
+	[![Screen capture of the MathML Cloud tool converting a math equation from ASCII into SVG, MML, and PNG formats.]({{{getRelativeUrl '/images/thumbs/400px-MathMLCloud.png' document.url}}})]({{{getRelativeUrl '/images/MathMLCloud.png' document.url}}})
+	<figcaption>
+		The MathML Cloud tool can convert math notation into readable text, SVG, MML, and PNG markup. The image shows the MathML interface converting an equation into different formats. Try MathMLCloud at <a href="https://www.mathmlcloud.org/" rel="nofollow" target="_blank" class="link-external">https://www.mathmlcloud.org/</a>
```
Comment (acheetham) [inline src/documents/Mathematics.html.md.handlebars]: The caption is a bit long. Is the first sentence necessary? And perhaps the link could be moved out of the caption?

File: src/documents/WebGamesAndSimulations.md [hunk 28]
```diff
@@ -0,0 +1,162 @@
...
+    * Describe any patterns to may help us understand the data.
+    * Summarize the trend / data pattern.
+* Also see:
+    * PSU’s chart accessibility guide - <a href="http://accessibility.psu.edu/images/charts/" rel="nofollow" target="_blank" class="link-external">http://accessibility.psu.edu/images/charts/</a>.
+    * W3C's guide on describing complex images - <a href="http://www.w3.org/WAI/tutorials/images/complex/" rel="nofollow" target="_blank" class="link-external">http://www.w3.org/WAI/tutorials/images/complex/</a>.
+
+If using a table of data:
+
+* Follow best practices for table formatting (i.e. proper use of table headers, no table nesting). <a href="https://www.webaccessibility.com/best_practices.php?technology_platform_id=8" rel="nofollow" target="_blank" class="link-external">See Data Table Best Practices at WebAccessibility.com</a>.
+* Any math, symbols, units, or acronyms defined. For additional information about making math accessible, see [Mathematics](Mathematics.html).
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: Change to "Define any math..."

File: src/documents/WebGamesAndSimulations.md [hunk 29]
```diff
@@ -0,0 +1,162 @@
...
+* Tables can be cumbersome to navigate (like with screen readers, small mobile displays, or screen magnifiers). Simplify and separate data tables if possible.
```
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: Change to "Simplify ... if possible. Tables can..."

File: src/documents/WebGamesAndSimulations.md (hunk 6 above)
Comment (acheetham) [inline src/documents/WebGamesAndSimulations.md]: Add a comma before "or"

File: src/documents/WebGamesAndSimulations.md [hunk 30]
```diff
@@ -0,0 +1,162 @@
...
+See: Poet Image Description Tool at <a href="http://diagramcenter.org/development/poet.html" rel="nofollow" target="_blank" class="link-external">http://diagramcenter.org/development/poet.html</a>.
+
+##  Cognitive
+
+Information should be presented in a clear concise manner, and where possible starting with general statements first with more detailed information following.
+
+* Instructions should be clear and concise, using a language that is appropriate.
+* If there is any possibility of confusion, make help readily available.
+* Avoid really dense content and interactions. Leverage semantic markup such as `<section>` elements with proper `<h1>` headings to help group content into navigable chunks.
+* Consider presenting information as needed, rather than presenting it all up front. Instructions may be better presented in context of the button or action (like a tooltip). This helps reduce cognitive load and gives information only when it's needed.
```
Comment (jhung) [inline src/documents/WebGamesAndSimulations.md]: I'm fine with either version. I will massage the "For example" point though. It seemed a bit wordy originally.
